scapy/fields.py

```diff
@@ -3224,13 +3227,15 @@ class ScalingField(_ScalingField,
        :param fmt: struct.pack format used to parse and serialize the internal value from and to machine representation # noqa: E501
        """
 
-class BitScalingField(_ScalingField, BitField):
+
+class BitScalingField(_ScalingField, BitField):  # type: ignore
```

Comment:
I had quite some trouble making BitScalingField work in the first place on both Python 2 and Python 3 so its typing will be mostly ignored.